templates/includes/sidebar.html

```diff
@@ -0,0 +1,14 @@
+<ul>
+  <li>
+    <a href = "#">Home page</a>
```

Comment:
The `href` attribute is set to `#`, which means it doesn't navigate anywhere. Consider using Django's `{% url 'index' %}` template tag to link to the home page.